app/controllers/internal_oai_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class InternalOaiController < ApplicationController
```

Comment:
Just wondering why we don't use inheritance here and override just endpoint solr_url.

Reply:
@dkinzer I spent a bunch of time trying to do a concern for these, but because of the way the `configure_blacklight` command works, it was messy and very confusing. Like, adding class methods each controller and calling those from the concern, which felt like a bad dependency direction.

There are actually multiple differences between the configurations, not just the solr url. So, the repository URL defined in the oai config, the repository name, etc. would all have to be overrriden, and I just didn't think that it will save anyone any time in the future trying to understand it if we went that route.

Reply:
OK, thanks.